src/content/basics/azure-architecture/index.md

```diff
@@ -1,47 +1,112 @@
 ---
 title: The Giant Swarm Azure Architecture
-description: Architecture Overview showing how Giant Swarm is set up on Microsoft Azure
-date: 2020-01-28
-weight: 60
+description: Description of the Azure Giant Swarm platform, how it looks like and which features offers
+date: 2020-06-10
+weight: 50
 type: page
 categories: ["basics"]
-last_review_date: 2020-01-28
+last_review_date: 2020-06-10
 ---
 
-# The Giant Swarm Azure Architecture
+The Giant Swarm Platform consists of various components. They can be categorized into three areas: infrastructure, operations, and applications.
 
-Giant Swarm's Architecture is split into two logical parts. One describes the Control Plane and the describes one or more Tenant Clusters. Among other services (for details see Service Architecture below), the control plane runs our [Azure Operator](https://github.com/giantswarm/azure-operator), which handles the full lifecycle management of Tenant Clusters.
+For managing all the infrastructure we run a control plane Kubernetes cluster per cloud and region where you want run your workloads. From that control plane you can spin up as many individual Kubernetes clusters as you want. Our operations team works to keep all cluster components healthy, while we release new versions with new features and patches. On top of that Giant Swarm offers a curated catalog with common Cloud Native tools that helps with monitoring, security or API management. Customers can leverage those while we carry the burden of maintain and keep them up to date.
 
-## Giant Swarm control plane
+When it comes to planning and designing your cluster architecture and its adaption to our infrastructure requirements, there are many moving parts to consider. Based on our experience with various customers over the last 6 years, we have gathered best practices and general advice to help with some of the initial critical decisions.
+
+## Control Plane
+
+As we are fully convinced of Kubernetes as a platform for building platforms, we built all our Control Plane based on a Kubernetes cluster. The initial deployment entails the creation of that Control Plane cluster in a defined cloud provider region. After the Control Plane cluster is ready we deploy all our automation taking advantage of Kubernetes primitives and using the same philosophy we advocate to our customers.
+
+Giant Swarm leverages the concept of [“Operators"](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to control all resources that clusters need as [“Custom Resources”](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). At the same time customers can also use the Kubernetes Control Plane API to [manage their clusters](https://docs.giantswarm.io/guides/creating-clusters-via-crs-on-aws/) and/or [applications](https://docs.giantswarm.io/basics/app-catalog/).
```

Comment:
This still in progress AFAIK, but we can keep it and docs reference when they are ready